parametric/README.md

```diff
@@ -0,0 +1,81 @@
+# APM library parametric tests
+
+The test fixtures in this submodule allow features of the APM libraries to be easily parameterized while still targeting
+each of the libraries.
+
+Example:
+
+```python
+@all_libs()
+@pytest.mark.parametrize("apm_test_server_env", [
+    {
+        "DD_SERVICE": "svc1",
+    },
+    {
+        "DD_SERVICE": "svc2",
+    }
+])
+def test_the_suite(apm_test_server_env: Dict[str, str], test_agent: TestAgentAPI, test_client: _TestTracer):
```

Comment:
IIRC, this could would be more readable with 

```python
    with test_client:
        span = start_span(name="web.request", resource="/users")
        span.set_meta("mytag", "value")
```

And `test_client.flush()` is done in `_TestTracer.__close__`